server/hpccloud/server/models/project.py

```diff
@@ -154,7 +154,7 @@ def delete(self, user, project):
 
         super(Project, self).remove(project)
 
-    def share(self, sharer, project, users, groups):
+    def share(self, sharer, project, users, groups, flags=[]):
         """
```

Comment:
@TristanWright Not sure I understand the question. The 'execute' permission is needed to run a particular taskflow type, for example PyFR. Not sure what it means to assign this permission to an instance of a project?